### ISSUE TITLE
Allow user to mark as ticket is resolved, so it's easy for rescue team

### ISSUE THREAD

**Issue by rabeesh** [IEEEKeralaSection/rescuekerala#254]
### Allow user ( who created the request ) to mark as ticket is resolved, once they are safe zone.

Eg:
1. when user requested for urgent help, and they got escaped by other alternative. Allow user to mark as ticket is "closed",  so it will remove the burden for rescue team.

2. Someone requested for food, they got food from other alternative. Let me mark as ticket as "closed".

**Comment by hari03** [comment]
Can be merged with issue #20

**Comment by hari03** [comment]
#20

**Comment by gokulan97** [comment]
So issue a request ID whenever someone requests something, and another form to close request, which accepts request ID and mobile number, used while issuing the request to close the request?

**Comment by rabeesh** [comment]
@gokulan97 

Yeah something like that. We have currently around 15000 request in the database. So you can close the request ID or mobile number would be helpful for rescue team. 

Also we need to "information  text" on request form page about this. 

> "Please try to close this request by your mobile number or request ID, once you are in safe zone.. This will helpful for rescue team".

**Comment by gokulan97** [comment]
I have zero prior experience in Django, it's taking some time for me. If anyone has expertise, please take this forward.
